demos/springboot-demo/vowelcount-service/src/main/java/io/netifi/proteus/demo/vowelcount/Application.java

```diff
@@ -31,14 +31,14 @@ public static void main(String... args) {
     }
 
 
-    @Bean
-    public ProteusConfigurer proteusConfigurer() {
-        return builder ->
-            builder.clientTransportFactory(address -> {
-                TcpClient client = TcpClient.create()
-                                            .addressSupplier(() -> address);
-                return TcpClientTransport.create(client);
-            });
-
-    }
+//    @Bean
+//    public ProteusConfigurer proteusConfigurer() {
+//        return builder ->
+//            builder.clientTransportFactory(address -> {
+//                TcpClient client = TcpClient.create()
+//                                            .addressSupplier(() -> address);
+//                return TcpClientTransport.create(client);
+//            });
+//
+//    }
```

Comment:
remove the commented code?

Reply:
DO we actually not need this? There have been occasions where @OlegDokuka you've commented stuff for testing but actually was necessary ;)

Reply:
It is a sample of how to customize proteus, maybe it should not be there at all like it breaks autoconfigured things. Just like a showcase. I can remove it at all or just make kind of identity showcase that log message "called"

Reply:
fixed